README.md

```diff
@@ -7,7 +11,7 @@ when you do a `git push`.
 Clone the repository:
 
 ```bash
-git clone https://github.com/lukasmartinelli/push-it.git
+git clone https://github.com/gynekolog/push-it.git
```

Comment:
It should still be the original repository. Then I will merge the PR :grin:

Reply:
You're right, I forgot change it too.

Reply:
I finally fix it. After 2.5 years... Better than never :-)